examples/django/README.md

```diff
@@ -0,0 +1,55 @@
+# Django Sample Application for Pybrake
+
+## About the application:
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
+```
+
+2. You must set both project_id & project_key in `settings.py`.
+
+To find your project_id and project_key from Airbrake account and replace it in below code in your project's `settings.py` file.
+
+```python
+AIRBRAKE = dict(
+    project_id=999999,                               # Insert your Project Id here
+    project_key='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',  # Insert your Project Key here
+    environment="test",
+    error_notifications=True,                        # False to disable error notification
```

Comment:
I don't think we want to advertise these options. I would remove lines 28-31. This should work out of the box already.

Reply:
Even `query_stats` and `queue_stats` cannot be disabled by the user?

Added those option over here based on https://github.com/airbrake/pybrake/blob/27c8906e8fe7e6e36faac3c267dbbf0f8184a15e/src/pybrake/notifier.py#L49

https://github.com/airbrake/pybrake/blob/27c8906e8fe7e6e36faac3c267dbbf0f8184a15e/src/pybrake/notifier.py#L50